cypress/support/fragments/settings/notes/settingNotes.js

```diff
@@ -0,0 +1,25 @@
+import uuid from 'uuid';
```

Comment:
Seem that noteTypes is a better name for the module

Reply:
done